chapters/tutorials/index.rst

```diff
@@ -0,0 +1,403 @@
+
+        targets = {
+            'covid-csv': {
+                'url': args.index_data_url,
+                'filename': os.path.join(args.workdir, 'dataset.csv'),
+            }
+        }
+
+        # download the data
+        download_data(targets, args.download_proxy, task_name='download covid-dataset')
+
+        f = (
+            Flow()
+                .add(name='MyTransformer', uses=MyTransformer)
+                .add(name='MyIndexer', uses=MyIndexer)
+                .plot('test.svg')
+        )
+
+        with f, open(targets['covid-csv']['filename']) as fp:
+            f.index(from_csv(fp, field_resolver={'question': 'text'}))
+
+If you run this, it should finish without errors. You won't see much yet because we are not showing anything after we index. But you should see a new folder created with the downloaded dataset:
+
+.. image:: res/downloaded_dataset.png
+   :width: 600
+
+To actually see something we need to specify how we will display it, and for our tutorial we will do so in our browser, so we need to add the following after indexing:
+
+.. code-block:: python
+
+        f.use_rest_gateway(args.port_expose)
+
+        url_html_path = 'file://' + os.path.abspath(
+            os.path.join(
+                os.path.dirname(os.path.realpath(__file__)), 'static/index.html'
+            )
+        )
+        try:
+            webbrowser.open(url_html_path, new=2)
+        except:
+            pass  # intentional pass, browser support isn't cross-platform
+        finally:
+            default_logger.success(
+                f'You should see a demo page opened in your browser, '
+                f'if not, you may open {url_html_path} manually'
+            )
+
+        if not args.unblock_query_flow:
+            f.block()
+
+For more information on what the Flow is doing, specially in `f.use_rest_gateway(args.port_expose)` and `f.block()` check our `cookbook <https://github.com/jina-ai/jina/blob/master/.github/2.0/cookbooks/Flow.md>`_
+
+Ok, so it seems that we have plentf of work done already. If you run this you will see a new tab in your browser open, and there you will have a text box ready for you to input some text. However, if you try to enter anything you won't have any results. This is because we are using very dummy Executors. Our `MyTransformer` and `MyIndexer` aren't actually doing anything. So far they only print a line when they are called. So we need real `Executors`.
+
+This has been already plenty of new information you've learned so far, so we won't go into `Executors` today, instead you can copy-paste the ones we are using for `this example <https://github.com/jina-ai/jina/blob/master/jina/helloworld/chatbot/executors.py>`_. The important part for you to understand is that it's here where you'll define exactly what you want your `Executors` to do. It can be something as simple as printing a line as we did today. Or something more complex as in the example.
```

Comment:
- today. **I**nstead
- "The important part for you to understand is that it's here where you'll define exactly what you want your `Executors` to do" -> "The important part to know is that all Executors' behavior is defined in `executors.py`.